File: WPF_GiamDinhBaoHiemYTe/Repos/Model/Patietn_XML_data/XML5.cs
using CommunityToolkit.Mvvm.ComponentModel;
using WPF_GiamDinhBaoHiem.Services.Interface;

namespace WPF_GiamDinhBaoHiem.Repos.Model
{
    public partial class XML5: ObservableObject, IHasStt
    {
        [ObservableProperty] private int id;
        [ObservableProperty] private string? ma_Lk;
        [ObservableProperty] private int? stt;
        [ObservableProperty] private string? dien_Bien_Ls;
        [ObservableProperty] private string? giai_DoAn_Benh;
        [ObservableProperty] private string? hoi_Chan;
        [ObservableProperty] private string? phau_Thuat;
        [ObservableProperty] private string? thoi_Diem_Dbls;
        [ObservableProperty] private string? nguoi_Thuc_Hien;
        [ObservableProperty] private string? du_Phong;
        [ObservableProperty] private bool isError;
    }
}

File: Backend/src/Repos/Models/XML_Model/XML5.js
const mongoose = require('mongoose');

const XML5Schema = new mongoose.Schema({
  id:               { type: Number, required: false },
  Ma_Lk:            { type: String, default: null },
  Stt:              { type: Number, default: null },
  Dien_Bien_Ls:     { type: String, default: null },
  Giai_DoAn_Benh:   { type: String, default: null },
  Hoi_Chan:         { type: String, default: null },
  Phau_Thuat:       { type: String, default: null },
  Thoi_Diem_Dbls:   { type: String, default: null },
  Nguoi_Thuc_Hien:  { type: String, default: null },
  Du_Phong:         { type: String, default: null }
}, {
  timestamps: true,
  strict: false
});

module.exports = mongoose.model('XML5', XML5Schema);
